utils/db.ts

```diff
@@ -226,69 +235,175 @@ export async function getUserById(id: string) {
   return res.value;
 }
 
+export async function getUserByLogin(login: string) {
+  const res = await kv.get<User>(["users_by_login", login]);
+  return res.value;
+}
+
+export async function getUserBySessionId(sessionId: string) {
```

Comment:
For sessions it would be good to attempt an eventually-consistent read first and fall back to strongly-consistent read if the first attempt returned not-found:

```ts
let res = await kv.get<User>(["users_by_session", sessionId], { consistency: "eventual" });
if(!res.value) {
  res = await kv.get<User>(["users_by_session", sessionId]);
}
return res.value;
```